### PR TITLE
[ESIMD] Add regression test for unused load.

### PR DESCRIPTION
**Pull request by kbobrovs**
Signed-off-by: Konstantin S Bobrovsky <konstantin.s.bobrovsky@intel.com>

### CODE REVIEW

File: SYCL/ESIMD/regression/unused_load.cpp [hunk 1]
```diff
@@ -0,0 +1,65 @@
+//==------ unused_load.cpp  - DPC++ ESIMD on-device regression test --------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// TODO enable on Windows
+// REQUIRES: linux && gpu
```
Comment (vladimirlaz) [inline SYCL/ESIMD/regression/unused_load.cpp:9]: The https://github.com/intel/llvm-test-suite/pull/77/files pr just enables ESIMD tets on Windows. So you can add this test test on Windows as well.

**Comment by vladimirlaz** [comment]
Could you please add links to tests. You can replace IDs of the jobs in th elinks below with the ones you have run:
http://llvm-ci.intel.com:8080/job/SYCL_CI/job/intel/job/Lin/job/LLVM_Test_Suite/1022/display/redirect
http://llvm-ci.intel.com:8080/job/SYCL_CI/job/intel/job/Lin/job/LLVM_Test_Suite_CUDA/956/display/redirect
http://llvm-ci.intel.com:8080/job/SYCL_CI/job/intel/job/Win/job/Test_Suite/231/display/redirect

**Review by vladimirlaz — CHANGES_REQUESTED** [review]
please address the comments

**Comment by kbobrovs** [comment]
@vladimirlaz - please review again, sorry for delay. Windows still can't be enabled due to driver issue.
Also, I did not understand this comment:
> Could you please add links to tests. You can replace IDs of the jobs in the links below with the ones you have run:

**Comment by vladimirlaz** [comment]
> @vladimirlaz - please review again, sorry for delay. Windows still can't be enabled due to driver issue.
> Also, I did not understand this comment:
> 
> > Could you please add links to tests. You can replace IDs of the jobs in the links below with the ones you have run:

You should run the tests in CI and put links to test jobs here. Please use format of links provided in my comment.

**Comment by kbobrovs** [comment]
> You should run the tests in CI and put links to test jobs here.

how to do that? /summary:run? I actually need to run this test only, as the code and other tests are not affected.

**Review by vladimirlaz — APPROVED** [review]

**Pull request merged**